labs-ios-starter/Feature-topic/CreateTopic/TopicController.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  TopicController.swift
+//  labs-ios-starter
+//
+//  Created by Kenny on 9/10/20.
```

Comment:
I don't know how you were able to compile and test with a invalid redeclaration of this class. Should the unique parts of this psuedo code be moved into the other file? It looks like one of your scratch pads for the TopicController is trying to get on the master branch.

Reply:
The project compiles because this file isn't part of it. This is the TopicController that I lost - its apparently still on disk, but not for long